netket/machine/jax.py

```diff
@@ -324,6 +324,137 @@ def JaxRbm(hilbert, alpha, dtype=complex):
     )
 
 
+def JaxMpsPeriodic(
+    hilbert, bond_dim, diag=False, symperiod=-1, dtype=complex, outdtype=complex
```

Comment:
Tom, this is already excellent, but we really need some documentation for the parameters etc
Would it be possible to add docstrings [like this](https://github.com/netket/netket/blob/v3.0/netket/machine/rbm.py#L39)? (at least for MpsPeriodic, not for the Layer if you don't have time)